### DISCUSSION TITLE
Add more tabs

### DISCUSSION THREAD

**Discussion by calabar77** [TeamAmaze/AmazeFileManager#3131] [Ideas]
Hi all and thanks for the good work. 
The application is "amazing" ;), even if it has some lacks for my needs. 

The application starts with two tabs, and I'm wondering if it is possible to add a third, a fourth, etc... tab. 
I'm accustomed to maintain opened more folders and use only two tabs ad the same time is limiting. 
A different icon for different type of folder would be useful too (for example a rhombus for smb folders instead than the dot). 

I take advantage of this topic to ask about some other features. Probably it is not the correct way to proceed but I want to avoid to flood the discussion area with many small requests. Anyway: 

- Above the content windows the app shows the path. If you click the path is transforms in clickable blocks that you can use to navigate back in the path. This is quite uncomfortable because it requires an additional click when you want to navigate, so I ask:  wouldn't it be better to show the clickable blocks by default and show the path (maybe editable) after a long click? 
- I've noticed that enabling "custom path" and "open file as a new activity" (I hope the translation is understandable) or maybe only one of them, when I change tab and then go back to the one where I used clickable blocks, the blocks are still there but have become unclickable. 
- Is it possible to have a special tab showing all the copied files/folders (or maybe the last ones) that allows me to paste them in a second time? At the moment when a copy some files immediately appears a bottom bar with the paste button, but if I make some other operation it vanishes forcing me to copy the source files again. Beside it is impossible to make more than one copy of a file without recopying it (or, at least, I haven't find a way to do it). 
- Is it possible to show the percentage (or some kind of diagram) of used space next to a "disk" name? 
- The experimental feature of select by dragging stop selecting when the window scrolls. 

Thanks!

**Comment by EmmanuelMess** [comment]
See #3023

**Comment by calabar77** [comment]
Thank you for the link. 

So it seems that having only two tabs is a design choice ant it is not possible to add new tabs in same way. 
I hope that this choice will change, it is very limiting for me. 

I've seen that a user produced some draft about a multi workspaces mode, but I've not understood if this solution will be implemented in a future release or it is simply a suggestion. I don't ask something so elaborated, but it would be surely welcome. 

I read about the "Open folders in new task" feature, but I suppose that it is another alternative solution about the same issue. 

About the other points above, have any sense to discuss them (even briefly) here?

**Comment by EmmanuelMess** [comment]
> wouldn't it be better to show the clickable blocks by default and show the path (maybe editable) after a long click?

Probably, there is an option to directly change the path by long clicking the bar in the settings.

> the blocks are still there but have become unclickable.

This is a bug, an issue should be opened.

> Is it possible to have a special tab showing all the copied files/folders (or maybe the last ones) that allows me to paste them in a second time?

If you can defend that many users want, or will use this, yes.

> Is it possible to show the percentage (or some kind of diagram) of used space next to a "disk" name? 

Please use the button on the drawer.

> The experimental feature of select by dragging stop selecting when the window scrolls.

This wouldn't allow users to select past the window's end.

**Comment by calabar77** [comment]
Thanks for the replay. I'm not sure I've fully understood all the answers, so I ask some clarification: 

1) I've see the option, but my need was not to edit the path but to directly access to the clickable blocks (without the extra click that I need to do now). It seems to me a layout improvement, is there any reason for the present choice? 
2) I suppose you mean I should open it. I'll take a look into this, I hope I will not need further help. 
3) I suppose that open a poll here is not the right way. ^___^ How to proceed? I thought it was a reasonable request of a basic (and useful) feature, I didn't thought I've to gather consents.  
4) Maybe I didn't explained well. Let's try an example. I've a list of 100 items, and I want to select 30 of them. The display of the smartphone can show only 10 of them at a time. So I start to touch the first one (keeping it pushed until the list enters in selection mode) and drag down to select the others but, when i reach the tenth, the list scrolls and the selection stops. To select the other items in the list I've to restart from the first unselected element (but often this don't work or don't work fine) or select them one by one (this works ever, but it is a bit annoying if the missing elements are many). 
I expected to select until I lift the finger instead, even if the list scrolls to reach the desired last element.
